src/account/mapper/managed-project.mapper.ts

```diff
@@ -16,21 +18,24 @@ export class ManagedProjectMapper {
             socialMedia: project.socialMedia ?? {},
             logo: project.logo
                 ? {
-                      key: project.logo.key,
+                      id: project.logo.id,
                       url: project.logo.url,
                   }
                 : {},
-            pictures: [],
+            images: [],
             teamName: project.teamName,
         } as AssociatedProject;
 
-        mappedProject.pictures = project.pictures?.map((picture: Picture) => {
-            return {
-                key: picture.key,
-                url: picture.url,
-            };
-        }) ?? [];
+        if (project.images.length > 0) {
```

Comment:
Got an exception here whilst testing locally. Needed to apply this: `project.images?.length`. If the project has no images associated the property is null

Reply:
normally all entries should have an empty array as default, maybe the migration wasn't executed correctly. I will add the check just to make sure we don't run into the problem again